cypress/utils/utils.ts

```diff
@@ -647,6 +647,26 @@ export function deleteApplicationTableRows(): void {
         });
 }
 
+export function deletePageApplicationTableRows(): void {
```

Comment:
There is a similar method called "deleteApplicationTableRows" in utils .

Can we merge these two ? 

There are three options in for page :
a) Select None
b) Select Page
c) Select All.

User can pass any of these as parameter and delete will work according to parameter passed .

Reply:
Solved.